### DISCUSSION TITLE
Probleme with connection

### DISCUSSION THREAD

**Discussion by lisliv** [maartentamboer/X-Touch-Mini-FS2020#122] [Q&A]
I have a probleme with connection. What should I do?

I tried to reinstall, disable everything that uses midi along the way, nothing helped.

![image](https://user-images.githubusercontent.com/117941274/201181510-a86920f6-3a48-4d92-9ea6-5c2982927ba4.png)

**Comment by krischi0511** [comment]
I think you mixed up the device numbers in the config.json. input device should be X-TOUCH MINI 1 and output device X-TOUCH MINI 0.

**Comment by GrimCultZero** [comment]
I have exactly the same problem and nothing is working. Wasted last money on X Touch Mini. Life is a bitch.

**Reply by Koseng** [reply]
You need to adjust your configuration in config.json. The numbers can be different on different systems.

Check what is printed in the console after "Midi Input Devices" and "Midi Output Devices" and what number is behind X-TOUCH MINI.

Then adjust in the config file:

```config.json```
"midi_input_device": "X-TOUCH MINI **0**",
"midi_output_device": "X-TOUCH MINI **1**",

**Comment by GrimCultZero** [comment]
> I have a probleme with connection. What should I do?
> 
> I tried to reinstall, disable everything that uses midi along the way, nothing helped.
> 
> ![image](https://user-images.githubusercontent.com/117941274/201181510-a86920f6-3a48-4d92-9ea6-5c2982927ba4.png)

I've found a solution to this: seems like you have 2 midi devices connected at the same time, as seen on log screen. One is XTouch Mini and the other one in your case is some "FW 410". Disconnect the other device, then plug out/plug in the Xtouch Mini - it will work. I had the same problem.
